cookbooks/cdo-varnish/test/integration/cloudfront/minitest/test_default.rb

```diff
@@ -0,0 +1,315 @@
+# Chef test suite for CloudFront distribution behavior.
+
+# Chef node attributes as constants
+DASHBOARD_PORT = 8080
+PEGASUS_PORT = 8081
+VARNISH_PORT = 80
+LOCALHOST = 'localhost'
+VARNISH_URL = "#{LOCALHOST}:#{VARNISH_PORT}"
+ENVIRONMENT = 'integration'
+CLOUDFRONT_URL = "https://#{ENVIRONMENT}-studio.code.org"
+
+# Setup the mock-server and ngrok
+puts 'setup'
+NGROK_PID = spawn('/usr/local/ngrok/ngrok start cdo --config=/home/kitchen/.ngrok2/ngrok.yml --log=stdout')
+PID = spawn('cd ~; java -jar mock.jar')
+# Don't start tests until wiremock is live.
+mock_started = false
+until mock_started
+  `sleep 1`
+  puts 'testing mock'
+  `curl -s -i #{LOCALHOST}:#{DASHBOARD_PORT}/start`
+  mock_started = $?.exitstatus == 0
+end
+puts 'setup finished'
+
+at_exit do
+  puts 'teardown'
+  `curl -s -X POST #{LOCALHOST}:#{DASHBOARD_PORT}/__admin/shutdown`
+  Process.wait PID
+  `kill -INT #{NGROK_PID}`
+  Process.wait NGROK_PID
+  puts 'teardown finished'
+end
+
+require 'json'
```

Comment:
Could you put these requires in a more conventional location at the top of the file?